app/models/studio.rb

```diff
@@ -1,3 +1,6 @@
 class Studio < ApplicationRecord
   has_many :movies
+  has_many :actors, through: :movies
```

Comment:
This is a good example of setting up a many-to-many by using existing relationships! Nice and neat.